src/m365/teams/commands/channel/channel-add.spec.ts

```diff
@@ -300,13 +335,43 @@ describe(commands.CHANNEL_ADD, () => {
         owner: 'john.doe@contoso.com'
       }
     });
+
     assert(loggerLogSpy.calledWith({
       "id": "19:d9c63a6d6a2644af960d74ea927bdfb0@thread.skype",
       "displayName": "Architecture Discussion",
       "membershipType": "private"
     }));
   });
 
+  it('creates shared channel within the Microsoft Teams team by team id', async () => {
+    sinon.stub(request, 'post').callsFake((opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/teams/6703ac8a-c49b-4fd4-8223-28f0ac3a6402/channels`) {
+        return Promise.resolve({
+          "id": "19:d9c63a6d6a2644af960d74ea927bdfb0@thread.skype",
+          "displayName": "Architecture Discussion",
+          "membershipType": "shared"
+        });
+      }
+      return Promise.reject('Invalid request');
+    });
```

Comment:
Could you refactor this stub to use `async/await` instead of `promise/then` please?

Reply:
You didn't had to refactor all tests to use `async/await` just the new tests were sufficient, but it's a good thing that this has been done now 😊